foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-APIManagementPolicySetDefinition.json

```diff
@@ -84,6 +84,328 @@
+                        "metadata": {
+                            "version": "1.0.0",
+                            "category": "API Management"
+                        },
+                        "parameters": {
+                            "effect": {
+                                "type": "String",
+                                "metadata": {
+                                    "displayName": "Effect",
+                                    "description": "Enable or disable the execution of the policy"
+                                },
+                                "allowedValues": [
+                                    "Deny",
+                                    "Audit",
+                                    "Disabled"
+                                ],
+                                "defaultValue": "Deny"
+                            },
+                            "evaluatedSkuNames": {
+                                "type": "Array",
+                                "metadata": {
+                                    "displayName": "API Management SKU Names",
+                                    "description": "List of API Management SKUs against which this policy will be evaluated."
+                                },
+                                "allowedValues": [
+                                    "Developer",
+                                    "Basic",
+                                    "Standard",
+                                    "Premium",
+                                    "Consumption"
+                                ],
+                                "defaultValue": [
+                                    "Developer",
+                                    "Premium"
+                                ]
+                            }
+                        },
+                        "policyRule": {
+                            "if": {
+                                "allOf": [
+                                    {
+                                        "field": "type",
+                                        "equals": "Microsoft.ApiManagement/service"
+                                    },
+                                    {
+                                        "field": "Microsoft.ApiManagement/service/sku.name",
+                                        "in": "[[parameters('evaluatedSkuNames')]"
+                                    },
+                                    {
+                                        "count": {
+                                            "field": "Microsoft.ApiManagement/service/privateEndpointConnections[*]",
+                                            "where": {
+                                                "allOf": [
+                                                    {
+                                                        "field": "[[split(concat(field('Microsoft.ApiManagement/service/privateEndpointConnections[*].id'), '//'), '/')[2]]",
```

Comment:
Let us avoid any policy that prevent cross-subscription scenarios for now (as mentioned in the storage PR).

Reply:
Removed